config/locales/en.yml

```diff
@@ -25,10 +25,16 @@ en:
       strip_insignificant_zeros: false
       unit: "$"
 
+  application:
+    home: "Home"
```

Comment:
Since these are links, what do you think about nesting them under `links`?
Or alternatively, change `application` to be `nav`...

Reply:
Good point, I think I will go with nav for the moment and review if necessary - done in sha: 1fd80f8